### PR TITLE
chore: remove ST v1

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- remove v1 -> v2 migration code
- remove all v1 libs 
- remove ldk old restore
- remove ldk recover screen
- remove AuthWidget
- remove old env variables
- add error message if user tries to scan slashauth url

I will squash commits after review

### Type of change

Refactoring

### Tests

No test

### QA Notes

Install prev version, add slashauth, upgrade, try to use slashauth. Widget should disapear

### CODE REVIEW

**Comment by Nuhvi** [comment]
@limpbrains I wonder if we should just merge this PR, test the performance of the app, to confirm that the DHT is causing most of the issues, then consider our options to add slashauth again in the light of that expirement.

**Comment by limpbrains** [comment]
Good idea, I will test it tomorrow

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "@react-native/typescript-config/tsconfig.json",
 	"compilerOptions": {
+		"jsx": "react",
```
Comment (pwltr) [inline tsconfig.json]: This option is set by `@react-native/typescript-config/tsconfig.json` with `"jsx": "react-native"

**Review by pwltr — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (limpbrains) [reply tsconfig.json]: Removed

File: src/utils/i18n/locales/en/slashtags.json [hunk 2]
```diff
@@ -88,15 +79,15 @@
...
 	"error_saving_contact": "Unable To Save Contact",
 	"error_saving_profile": "Unable To Save Profile",
 	"error_delete_contact": "Unable To Delete Contact",
 	"error_sdk_title": "Please Restart Bitkit",
 	"error_sdk_msg": "An essential component is not working properly. Please restart Bitkit.",
 	"error_pay_title": "Unable To Pay Contact",
-	"error_pay_empty_msg": "There is no payment data available for this contact."
+	"error_pay_empty_msg": "There is no payment data available for this contact.",
+	"auth_depricated_title": "Deprecated",
+	"auth_depricated_msg": "Slashauth is deprecated. Please Bitkit Beta."
```
Comment (pwltr) [inline src/utils/i18n/locales/en/slashtags.json]:
```suggestion
	"auth_depricated_msg": "Slashauth is deprecated. Please use Bitkit Beta."
```

**Review by pwltr — CHANGES_REQUESTED** [review]

**Review by pwltr — APPROVED** [review]
Tested and LGTM

**Pull request merged**